diff --git a/js/SaveInterface.js b/js/SaveInterface.js
index 2b8f36e47c54beef53ccfdc43a8626b44187874b..9521b864465c3e666591e5f38d4d5864d8bf7758 100644
--- a/js/SaveInterface.js
+++ b/js/SaveInterface.js
@@ -84,7 +84,7 @@ function SaveInterface(PlanetInterface) {
             if (window.isElectron == true) {
                 filename = defaultfilename;
             } else {
-                filename = prompt("Filename:", defaultfilename);
+                filename = defaultfilename;
             }
         } else {
             if (fileExt(defaultfilename) != extension) {
@@ -484,6 +484,7 @@ function SaveInterface(PlanetInterface) {
 
     this.init = function() {
         this.timeLastSaved = -100;
+        /*
         window.onbeforeunload = function(e) {
             if (
                 this.PlanetInterface !== undefined &&
@@ -516,5 +517,6 @@ function SaveInterface(PlanetInterface) {
                 e.returnValue = "";
             }
         }.bind(this);
+        */
     };
 }
